doc/design/engine.xml

```diff
@@ -484,6 +484,17 @@
     </listitem>
    </varlistentry>
 
+   <varlistentry>
+    <term><literal>.relpath</literal></term>
+    <listitem>
+     <para>
+
+      The path to the file relative to the root SConstruct file's directory.
+
+     </para>
+    </listitem>
+   </varlistentry>
+
```

Comment:
We don't usually update these files, the design doc is woefully out of date.  That's more an FYI, if @bdbaddog doesn't object we can leave this here.